utils/lib/rr_graph/tracks.py

```diff
@@ -225,16 +227,37 @@ def is_wire_adjacent_to_track(self, idx, coord):
             assert False, track
 
     def get_tracks_for_wire_at_coord(self, coord):
-        """Returns which track indicies and direction a wire at a coord can
-        be connected too.
+        """Returns map of direction to track indicies for valid connections.
+
+        There may be multiple connections to tracks at a particular coordinate,
+        this method only returns a possible connection in each direction this
+        this track.
+
+        Parameters
+        ----------
+        coord : (int, int)
```

Comment:
FYI - We should use [typing annotation](https://mypy.readthedocs.io/en/latest/cheat_sheet_py3.html) rather than doing it in the docstring (where it works).